scripts/version/push.sh

```diff
@@ -1,23 +1,18 @@
 #!/bin/sh
 setup_git() {
-  git config --global user.email "travis@travis-ci.org"
-  git config --global user.name "Travis CI"
+  git config --global user.email "litmusbotci@gmail.com"
+  git config --global user.name "litmus[bot]"
   git remote set-url origin  https://${GITHUB_TOKEN}@github.com/litmuschaos/community-charts.git > /dev/null 2>&1
```

Comment:
should we change or remove this?

Reply:
We need to configure this. We can use any id or name but without configuring it will not commit the changes:
I removed `git config` and ran checkout the issue [here](https://github.com/uditgaurav/test-chaos-charts/runs/1599939543#step:6:24)

Reply:
Can be removed!